SimTKcommon/include/SimTKcommon/internal/ClonePtr.h

```diff
@@ -202,7 +207,11 @@ template <class T> class ClonePtr {
         other.reset(p);
         reset(otherp);
     }
-     
+
+    DEPRECATED_14("use get() instead")
+    const T* getPtr()  const  { return get(); }
+    DEPRECATED_14("use upd() instead")
+    T* updPtr() { return upd(); }
```

Comment:
> By the way, Doxygen has a way to mark a method as deprecated

I tried that and found it unsatisfactory. All it actually does is create a new paragraph with heading "Deprecated", which is not very useful. Better would be something grayed out, marked with a distinctive logo, or something like that.